frontend/src/scenes/sessionRecordings/SessionPlayerDrawer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { Drawer } from 'lib/components/Drawer'
+import { SessionsPlay } from 'scenes/sessions/SessionsPlay'
+import { useActions } from 'kea'
+import { ArrowTopLeftOutlined } from 'lib/components/icons'
+import { sessionRecordingsTableLogic } from './sessionRecordingsTableLogic'
+
+interface SessionPlayerDrawerProps {
+    personIds?: string[]
+    isPersonPage?: boolean
+}
+
+export function SessionPlayerDrawer({ personIds, isPersonPage = false }: SessionPlayerDrawerProps): JSX.Element {
```

Comment:
Nit: It seems like not much here will change other than the logic and props passed in. Is there a way to reuse `SessionPlayerDrawer` [here](https://github.com/PostHog/posthog/blob/d87591167eb5f31a7776e3a3442efdcc625316ab/frontend/src/scenes/sessions/SessionsView.tsx#L45-L45) by refactoring it into a dumb component?